peps/pep-0725.rst

```diff
@@ -414,7 +457,7 @@ SciPy 1.10:
     ]
     host-requires = [
       "virtual:interface/blas",
-      "virtual:interface/lapack",  # >=3.7.1 (can't express version ranges with PURL yet)
+      "dep:interface/lapack@>=3.7.1",
```

Comment:
After this change there's a mix of `virtual:` and `dep:interface` in the document. Why would we allow both?

Reply:
> I really prefer having a single scheme.

I think you are right here. `dep:interface` seems fine to me, and nicer than a separate `virtual:`.

Reply:
Fixed this now.